docs/md/person-name-fields.md

```diff
@@ -0,0 +1,26 @@
+# Person name fields
+
+Janeway has ten or so fields related to names of people. They are spread across the `Account` and `FrozenAuthor` models.
+
+These fields are generally subjected to plain text validation to ensure no markup is injected, since they are included in some strings that are marked safe and parsed as HTML.
+
+## Account name fields
+
+We have three fields for first, middle, and last name. Separate fields are useful needed for producing citations, consistent alphabetization, and downstream metadata requirements from ORCID, Crossref, JATS.
+
+However, there are well-documented problems with separate name fields and making them required fields. See [this W3C article about different conventions](https://www.w3.org/International/questions/qa-personal-names), and this [list of falsehoods programmers believe about names](https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/). If possible, future Janeway development on name fields should make them more flexible.
```

Comment:
I don't think this should be in the docs, please move this to an issue on changing how we handle name fields.